travel_platform/travel_app/views.py

```diff
@@ -10,36 +11,24 @@ def index(request):
     return render(request, 'travel_app/main.html')
 
 
+@cache_page(86400)
```

Comment:
The caching key looks like url with GET parameters and response is its value is it suitabale for us??